int8 MB_pole
float64 x
float64 y
float64 x_sigma
float64 y_sigma
time stamp
